src/deckardcain.js

```diff
@@ -1,5 +1,6 @@
 export const API_BLUEPRINT_HEADER = /^[\uFEFF]?(((VERSION:( |\t)2)|(FORMAT:( |\t)(X-)?1A))\s*?([\n\r]{1,2}|$))/i;
-export const API_BLUEPRINT_RESPONSE = /\+\s+(?:response|request)\s+\d{3}/i;
+export const API_BLUEPRINT_RESPONSE = /[-\*\+]\s+(response|request|attributes?)/i;
```

Comment:
I was pretty defensive designing the original RE as matching the string against `+ Response` was pretty much just wishful thinking heuristic. I felt that `+ Response` is so prevalent in API Blueprint documents that assuming any content containing it as API Blueprint can do no harm.

This change makes it a lot more permissive, which can lead to a lot more false positives, but then I guess the parser should be the ultimate judge... In the context of API description documents, I believe this might still be feasible as OpenAPI documents will be detected before this is executed and there are basically no other competing formats. This just defends parsing random plaintext files.

You should change the variable name though: `API_BLUEPRINT_RESPONSE` doesn't really describe its current content.

Reply:
There's a particular test to ensure that too:

```js
  describe('Something that is Swagger but contains Blueprint stuff', function() {
    const sources = [
      '{"swagger":"2.0","content":"--- Ha ha ha ---"}',
      '{"swagger":"2.0","content":"+ Response 200"}',
    ];

    it('is identified as Swagger', function() {
      sources.forEach((source) => {
        assert.equal(identify(source), 'application/swagger+json');
      });
    });
  });
});
```